appium-cookies/lib/cookies.js

```diff
@@ -0,0 +1,75 @@
+/*
+ * derived from jQuery Cookie Plugin v1.4.1
+ * https://github.com/carhartl/jquery-cookies
+ */
+
+ //needed to communicate/translate between JSONWire cookies and regular JS cookies 
+
+import _ from 'lodash';
+
+export default class Cookies {
+
+  constructor (cookieString) {
+    this.cookieString = cookieString;
+  }
+
+  //parses the value if needed and converts the value if a converter is provided
+  convertCookie(value, converter) {
+    let parsedValue;
+
+    if (value.indexOf(`"`) === 0) {
+      value = value.slice(1, -1).replace(/\\"/g, `"`).replace(/\\\\/g, `\\`);
+    }
+    try {
+      parsedValue = decodeURIComponent(value.replace(/\+/g, ` `));
+    } catch (e) {}
```

Comment:
In what cases would we have an exception here? Should we log it at the very least?